docs/api/graphql/overview.md

```diff
@@ -12,9 +12,10 @@ For general guidance and info you need to know about the APIs provided by Vega,
 :::
 
 ## Queries
+
 Here is a simple query to fetch the current block height and Vega time from a data node:
 
-```
+```json
```

Comment:
**nitpick (non-blocking)**:  should be `graphql` rather than `json` as it's invalid JSON

Reply:
```suggestion
```graphql
```